index.html

```diff
@@ -2,316 +2,349 @@
-				<h1 class="card-header font-style">
-					Tonic
-				</h1>
-				<section class="card-description">
-					<ul class="font-style">
-						<li class="card-list-title"> CANOPY</li>
-						<li class="Job-title_and_year">
-							Back End Dev
-						</li>
-						<li class="Job-title_and_year">
-							2022
-						</li>
-					</ul>
-				</section>
-				<p class="notice font-style">
-					A daily selection of privately personalized reads; no accounts or sign-ups required.
-				</p>
-				<ul class="programming-languages font-style">
-					<li>html</li>
-					<li>css</li>
-					<li>javaScript</li>
-				</ul>
-				<button class="project-button button" type="button">See Project</button>
-			</section>
-		</section>
-		<section class="card">
-			<img class="snapshoot-style" src="images/Snapshoot Portfolio4.svg" alt="Snapshoot portfolio image">
-			<section>
-				<h1 class="card-header font-style">
-					Multi-Post <br> Stories
-				</h1>
-				<section class="card-description">
-					<ul class="font-style">
-						<li class="card-list-title"> CANOPY</li>
-						<li class="Job-title_and_year">
-							Back End Dev
-						</li>
-						<li class="Job-title_and_year">
-							2022
-						</li>
-					</ul>
-				</section>
-				<p class="notice font-style">
-					A daily selection of privately personalized reads; no accounts or sign-ups required.
-				</p>
-				<ul class="programming-languages font-style">
-					<li>html</li>
-					<li>css</li>
-					<li>javaScript</li>
-				</ul>
-				<button class="project-button button" type="button">See Project</button>
-			</section>
-		</section>
-	</div>
-	<div class="about-me">
-		<section>
-			<h2 class="about-me-heading font-style">About <br> Myself</h2>
-			<p class="description font-style">
-				I’m a software developer! I can help you build a product , feature or
-				website Look through some of my work and experience! If you like what
-				you see and have a project you need coded, don’t hestiate to contact
-				me.
-			</p>
-		</section>
+        </section>
 
-		<section>
-			<h4 class="connect font-style">LET'S CONNECT</h4>
-			<ul class="socials">
-				<li>
-					<a class="social_medias twitter" href="https://twitter.com/FentahunAgmas">
-						<img src="images/Vector (1).svg" alt="twitter logo">
-					</a>
-				</li>
+        </section>
+        <section class="card second-card">
+            <img class="snapshoot-style" src="images/Snapshoot Portfolio2.svg" alt="Snapshoot portfolio image">
+            <section>
+                <h2 class="card-header font-style on-small">
+                    Multi-Post <br> Stories
+                </h2>
+                <h2 class="card-header font-style on-large">
+                    Uber Navigation
+                </h2>
+                <section class="card-description">
+                    <ul class="font-style">
+                        <li class="card-list-title on-small"> CANOPY</li>
+                        <li class="card-list-title on-large"> Uber</li>

```

Comment:
- [ ] You did good a job on this milestone ,however their are some little changes you only need to amend.
- [ ] I would kindly suggest you to use a html semantic tag on each of your sections ,your page consist of toolbar,header,working section ,About me section and contact me ,kindly arrange your page section by section with the section tag ,About myself should have its own sections .working sections should hold its own section tag also and conatct me should have its own section ,this will make your code easy to read and professional please fix this .